recipes/pythran/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "pythran" %}
+{% set version = "0.8.5" %}
+{% set sha256 = "92f67171d03ba3dbed9da639c35aad2fe0982c183f0c4459dcfddd621f69795e" %}
+{% set variant = "openblas" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  skip: true  # [win]
+  features:
+    - blas_{{ variant }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - gmp
+    - numpy 1.13.*
+    - pytest-runner
+  run:
+    - {{ compiler('cxx') }}
```

Comment:
Shouldn't the choice of compiler be determined by the dependent packages?

Reply:
It makes sense to install pythran on its own. Is there a way to force the user to install a compiler with it?

Reply:
There's no way to force the user to install a compiler with it. That's fine though. Cython doesn't force the user to install a compiler.